docs/source/collect/collectors.md

```diff
@@ -28,11 +28,10 @@ The following properties are supported on all collectors:
 #### `collectorName`
 
 Optionally, a collector can specify the `collectorName` property.
-This controls the exact path where the collector will be stored in the support bundle.
-If this property is not included, a generated path name that's relevant to the collector type and properties will be used.
+In some collectors this controls the path where result files will be stored in the support bundle.
```

Comment:
Details are under each collector doc. Only used by http, run, redis, mysql, and postgres collectors. Some collectors have a `name` field that does the same thing.